context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+The programming language is for people that want to be able to create digital art without learning how to render it in a different programming language. In ContextFree, the only code that the user needs to write involves the image that they produce. The user does not need to learn about graphics libraries or additional features of a more complex language. In general, ContextFree is optimal for someone who wants to create graphical images who has a basic understanding of computer science.
```

Comment:
^ Agreed. A good way to think about the intended user is to consider the how this language differs from those that might appear similar. So while it is easy to draw shapes without having to render them, it really isn't easy to draw shapes in specific locations. It isn't intuitive to draw a normal life-like picture with recursion. So you have to consider what type of image the user wants to draw. 

What is different about contextFree is that it has randomness and recursion built into it. Compared to other languages that have drawing libraries, this focuses on recursion and a grammar like building of images. So the intended user most likely has a need for these features, since there are a lot of languages that support drawing images with basic programming.